Dockerfile.d/test-integration-rootless.sh

```diff
@@ -26,7 +26,7 @@ if [[ "$(id -u)" = "0" ]]; then
 	fi
 
 	: "${WORKAROUND_ISSUE_622:=}"
-	if [[ "$WORKAROUND_ISSUE_622" = "1" ]]; then
+	if [[ "$WORKAROUND_ISSUE_622" != "" ]]; then
```

Comment:
This breaks `WORKAROUND_ISSUE_622=0` but I guess we do not need to keep compatibility for this workaround